public/locales/en-US/grafana.json

```diff
@@ -144,9 +154,67 @@
       "source": "Source",
       "target": "Target"
     },
+    "navigation-form": {
+      "add-button": "Add",
+      "back-button": "Back",
+      "next-button": "Next",
+      "save-button": "Save"
+    },
     "page-content": "To enable Correlations, add it in the Grafana config:",
     "page-heading": "Correlations are disabled",
-    "sub-title": "Define how data living in different data sources relates to each other. Read more in the <2>documentation<1></1></2>"
+    "query-editor": {
+      "control-rules": "The selected target data source must export a query editor.",
+      "data-source-text": "Please select a target data source first.",
+      "data-source-title": "No data source selected",
+      "error-text": "The selected data source could not be loaded.",
+      "error-title": "Error loading data source",
+      "loading": "Loading query editor...",
+      "query-description": "Define the query that is run when the link is clicked. You can use <2>variables</2> to access specific field values.",
+      "query-editor-title": "Data source does not export a query editor.",
+      "query-label": "Query"
+    },
+    "source-form": {
+      "control-required": "This field is required.",
+      "description": "You have used following variables in the target query: {variableList({variables})}<2></2>A data point needs to provide values to all variables as fields or as transformations output to make the correlation button appear in the visualization.<4></4>Note: Not every variable needs to be explicitly defined below. A transformation such as <7>logfmt</7> will create variables for every key/value pair.",
```

Comment:
I think this might be stale and you may need to regenerate translations again.

It produces the following:

<img width="1123" alt="Screenshot 2023-10-11 at 14 18 50" src="https://github.com/grafana/grafana/assets/745532/c2598ae3-8b63-4366-8320-d5de94bd1ef2">

Regenerating translations should create something like:

`"description": "You have used following variables in the target query: <1></1><2></2>A data point needs to(...)",
`

Reply:
Hey @khushijain21! I think this still needs to be re-generated. Are you planning to continue working on this PR?

Reply:
Hello @ifrost, just did it, can you check?

Reply:
any update?